content/posts/2021/pangeo-goes-live/index.md

```diff
@@ -0,0 +1,48 @@
+
+## What is the Pangeo community?
+
+Pangeo has pioneered a new model in using open source and cloud-agnostic infrastructure to support scientific research in the cloud.
+
+The first Pangeo cloud JupyterHub (pangeo.pydata.org; now defuct) was deployed for the [2017 American Meteoroligical Society Meeting](https://annual.ametsoc.org/2017/); since then, the Pangeo community has iterated through several different versions of prototype cloud-based hubs.
+This allowed for many new workflows that enabled a more open and collaborative pathway to doing world class research, and included access to datasets and computational resources that were previously unattainable.
+Pangeo achieved this by working in partnership with open source communities and building technology that leveraged modular open source components for their platform.
+
+In the last several years, Pangeo have built a thriving community of practice around this infrastructure.
+However as the community has grown, so has the need for more reliable and dedicated operational and developmental support since parts of the Pangeo stack require dedicated expertise and attention to managed.
+Modern scalable cloud infrastructure is one example of this. Maintaining a complex JupyterHub with many users is a difficult task, and has required significant resources from the Pangeo Project up to this point.
+
+## The Pangeo-2i2c Partnership
+
+[2i2c](https://2i2c.org) is a non-profit team that develops and operates cloud infrastructure for interactive computing workflows.
+We have extensive experience in Jupyter workflows in the cloud and a long history of contributions to projects in this ecosystem.
+We have built a cloud deployment management system that allows us to centralize ad configure the deployment of many independent JupyterHubs, empowering communities to leverage the same infrastructure (and team!) for JupyterHubs running in the cloud.
+
+Similarly to Pangeo, all of 2i2c's core infrastructure is cloud- and vendor-agnostic, and follows a model of building open source tools and giving back to those communities.
+Our partnership with Pangeo began through 2i2c's core competency in these areas and the similarity between the two project's technical stacks.
+
+We've begun a partnership wherby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.
+This approach allows each community to focus on it's core strengths: Pangeo will continue to grow an open community and scientific software ecosystem around geospatial analytics, and 2i2c will oversee the development and operations of the core cloud infrastructure stack that powers Pangeo's workflows.
+
+We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:
```

Comment:
From the HackMD:

> Would be nice to have a final section hinting at some of the plans we have for the future.

I'm not sure we've decided this yet? I guess the next step is to migrate the GCP Binder? Is that the kind of work that we should hint at here?

Reply:
I think that in this case we can just leave it relatively hand-wavy and point to that Discourse post to give folks an idea of what might be done

Reply:
I added that discourse post a little further up the previous paragraph for flow.